### PR TITLE
Improve code quality

### PR DESCRIPTION
**Pull request by ys112**
- Remove unused code found.
- Add more tests for import command.
- Refactor of code.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ImportCommand.java
Comment (ChrisHo1341) [inline src/main/java/seedu/address/logic/commands/ImportCommand.java:1]: LGTM

File: src/main/java/seedu/address/model/applicant/Applicant.java
Comment (ChrisHo1341) [inline src/main/java/seedu/address/model/applicant/Applicant.java:1]: LGTM

File: src/main/java/seedu/address/model/applicant/Role.java
Comment (ChrisHo1341) [inline src/main/java/seedu/address/model/applicant/Role.java:1]: LGTM

File: src/test/java/seedu/address/logic/parser/ImportCommandParserTest.java
Comment (ChrisHo1341) [inline src/test/java/seedu/address/logic/parser/ImportCommandParserTest.java:1]: LGTM

File: src/test/java/seedu/address/logic/commands/ImportCommandTest.java
Comment (ChrisHo1341) [inline src/test/java/seedu/address/logic/commands/ImportCommandTest.java:1]: LGTM

**Review by ChrisHo1341 — APPROVED** [review]

File: src/main/java/seedu/address/logic/commands/ImportCommand.java
Comment (YuvBindal) [inline src/main/java/seedu/address/logic/commands/ImportCommand.java:1]: LGTM

File: src/main/java/seedu/address/model/applicant/Applicant.java
Comment (YuvBindal) [inline src/main/java/seedu/address/model/applicant/Applicant.java:1]: LGTM

File: src/main/java/seedu/address/model/applicant/Stage.java
Comment (YuvBindal) [inline src/main/java/seedu/address/model/applicant/Stage.java:1]: LGTM

File: src/test/data/JsonAddressBookStorageTest/validApplicantHRConnect.json
Comment (YuvBindal) [inline src/test/data/JsonAddressBookStorageTest/validApplicantHRConnect.json:1]: LGTM. Nice JSON structure

File: src/test/java/seedu/address/logic/parser/ImportCommandParserTest.java
Comment (YuvBindal) [inline src/test/java/seedu/address/logic/parser/ImportCommandParserTest.java:1]: LGTM

**Review by YuvBindal — COMMENTED** [review]
Overall solid. Looks good to merge to the main branch

**Pull request merged**